### PR TITLE
issue #15

### PR DESCRIPTION
**Pull request by akshatshukla13**
- Installed Prettier as a dev dependency
- Created .prettierrc for custom formatting rules
- Added .prettierignore to exclude unnecessary files
- Updated package.json with format script
- Integrated Prettier with ESLint to avoid conflicts

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    extends: [
+      "plugin:prettier/recommended"
```
Comment (zetareticoli) [inline .eslintrc.js:3]: > - Integrated Prettier with ESLint to avoid conflicts

Is this the related commit?

**Review by zetareticoli — COMMENTED** [review]
Have you tried to run prettier on library files to optimize code?

File: .eslintrc.js (hunk 1 above)
Comment (akshatshukla13) [reply .eslintrc.js:3]: This will disables the conflicting ESLint rules.

**Comment by akshatshukla13** [comment]
Anything causing errors or problems, let me know!
